dev-quick-start.md

```diff
@@ -1,7 +1,9 @@
-# K8ssandra Developer - Quick Start
+# K8ssandra Chart Development Quick Start
+
+*Before diving into chart development we recommend you start with the broader [K8ssandra Development Guide](dev-guid.md).*
```

Comment:
typo `dev-guid.md` should be `def-guide.md`

Reply:
Fixed, thanks for the catch.